app/code/Magento/Quote/etc/db_schema.xml

```diff
@@ -352,6 +352,8 @@
                 comment="Super Product Id"/>
         <column xsi:type="int" name="parent_product_id" padding="10" unsigned="true" nullable="true" identity="false"
                 comment="Parent Product Id"/>
+        <column xsi:type="smallint" name="store_id" padding="5" unsigned="true" nullable="true" identity="false"
```

Comment:
Unfortunately, we can't add a foreign key to store table due to [Split database performance solution](https://devdocs.magento.com/guides/v2.3/config-guide/multi-master/multi-master.html). This solution provides an ability to move quote related tables to a separate database and with current changes, we will have the fatal error due to missing table.

Reply:
@sidolov , foreign key removed